content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+pre-configured beverage, to be brought to them while they wait. This application is ideal for
+businesses that do not have someone working at a reception desk, or perhaps do not have a lobby or
+waiting area available to guests and visitors.
+
+Configuration
+=============
+
+The first item to be configured to use the *Frontdesk* application is the station or stations,
+followed by the drinks that are offered.
+
+Stations
+--------
+
+A **Station** can be thought of as any location where someone can sign-in and wait for an employee.
+This is typically some form of waiting room, such as a lobby. Each station has a kiosk where
+visitors check in. There is a minimum of one station that must be configured to use the *Frontdesk*
+application, but there is no limit to how many stations that can created and configured.
+
+To create a station, navigate to :menuselection:`Frontdesk application --> Configuration -->
+Stations` and click :guilabel:`New`. A blank frontdesk form loads. Enter the following information
+on the form:
+
+- :guilabel:`Frontdesk Name`: enter the name for the specific Frontdesk location. This should be
+  short and identifiable, such as `Reception Desk` or `Main Lobby`. This field is required in order
+  to create a station.
+- :guilabel:`Responsibles`: select the person or persons who are alerted when a visitor checks in
+  using at specific frontdesk. Multiple selections can be entered. This field is required in order
+  to create a station.
+- :guilabel:`Kiosk URL`: after both the :guilabel:`Frontdesk Name` and :guilabel:`Responsibles` are
+  populated, the form can be saved manually. Click the :guilabel:`Save Manually` icon (a cloud with
+  an up arrow inside it) and the :guilabel:`Kiosk URL` field populates. This URL is one way the
+  frontdesk kiosk is accessed. To access the kiosk, click the :guilabel:`Copy` button at the end of
+  the URL, and navigate to that URL in a web browser. This URL opens that specific stations'
+  frontdesk sign-in page.
+
+Options tab
+~~~~~~~~~~~
+
+.. _frontdesk/host:
+
+- :guilabel:`Host Selection`: if the visitor is attending a meeting, this option allows the visitor
+  to select the meeting host from a presented list, and notify that individual.
+- :guilabel:`Authenticate Guest`: if additional information is required when a guest checks-in,
+  enable this option, and then select which of the following are required:
+
+  - :guilabel:`Email`: select whether the guest's email address is :guilabel:`Required`,
+    :guilabel:`Optional`, or if the information is not requested at all (:guilabel:`None`).
+  - :guilabel:`Phone`: select whether the guest's phone number is :guilabel:`Required`,
+    :guilabel:`Optional`, or if the information is not requested at all (:guilabel:`None`).
+  - :guilabel:`Organization`: select whether the guest's organization is :guilabel:`Required`,
+    :guilabel:`Optional`, or if the information is not requested at all (:guilabel:`None`).
+
```

Comment:
Should this extra line be here between `Optional` and `Theme`? Or is it all the same list?

Reply:
Yes, because there's a weird issue when you need to indent, I noticed you have to add a space before and after a section you want indented, or it's messed up. I remember trying a lot of different ways to get it to work but this seems to do it!